qubekit/nonbonded/lennard_jones.py

```diff
@@ -298,3 +331,73 @@ def _calculate_b_prime(
             non_bonded_forces[atom.atom_index] = (sigma, epsilon)
 
         return non_bonded_forces
+
+
+class LennardJones612Delta(_LennardJonesBase):
+    """
+    An experimental QM to MM atom mapping function which allows flexibility in the scaling of the volume ratio when
+    calculating sigma not for production yet, also does not allow for polar H transfer.
+    """
+
+    type: Literal["LennardJones612Delta"] = "LennardJones612Delta"
+
+    alpha: float = Field(
+        default=1.0,
+        description="The amount by which the aim/free volume ration should be scaled.",
+    )
+    delta: float = Field(
+        1 / 3,
+        description="The amount by which the volume ratio should be scaled when calculating sigma",
+    )
+
+    def _run(self, molecule: "Ligand", *args, **kwargs) -> "Ligand":
+        """Calculate the sigma and epsilon based on the delta scaling relation."""
+
+        # check we can parameterize the molecule
+        self.check_element_coverage(molecule=molecule)
+
+        # calculate sigma and epsilon
+        non_bonded_data = self._calculate_sig_eps(molecule=molecule)
+        # update the parameters
+        LennardJones612Delta._update_molecule_parameters(
+            non_bonded_data=non_bonded_data, molecule=molecule
+        )
+        return molecule
+
+    def _calculate_sig_eps(self, molecule: "Ligand") -> Dict[int, Tuple[float, float]]:
+        """
+        Calculate the sigma and epsilon parameters for each atom and return a dict to update the molecule with.
+        """
+        sigma_and_eps = {}
+
+        for atom in molecule.atoms:
+            atomic_symbol, atom_vol = atom.atomic_symbol, atom.aim.volume
+
+            # check if we have a separate Rfree for polar H stored as X
+            if "X" in self.free_parameters and atomic_symbol == "H":
+                atomic_symbol = LennardJones612Delta._get_h_type(
+                    h_atom=atom, molecule=molecule
+                )
+
+            volume_ratio = atom_vol / self.free_parameters[atomic_symbol].v_free
+            sigma = (
+                (2 ** (5 / 6))
+                * (volume_ratio**self.delta)
+                * self.free_parameters[atomic_symbol].r_free
+            )
+            # convert from angstroms to nanometers
+            sigma *= constants.SIGMA_CONVERSION
+            # create beta from delta
+            beta = 2 - (6 * self.delta)
+            epsilon = (
+                self.alpha
+                * (volume_ratio**beta)
+                * self.free_parameters[atomic_symbol].b_free
+            )
+            epsilon /= 128 * self.free_parameters[atomic_symbol].r_free ** 6
+            # convert from Hartrees to KJ / mol
+            epsilon *= constants.EPSILON_CONVERSION
+
+            sigma_and_eps[atom.atom_index] = (sigma, epsilon)
+
+        return sigma_and_eps
```

Comment:
@djcole56 Please can you double-check the maths and conversions here?

Reply:
Looks good to me. Note that you've introduced beta here, which has a different meaning to the beta used in eq 11 in Ringrose et al. so a different variable name might be better.